request/parser.go

```diff
@@ -14,6 +14,13 @@ import (
 	"github.com/buildwithgrove/path/relayer"
 )
 
+const (
+	// TODO_DISCUSS: should we create a common errors package for all errors?
+	// TODO_IMPROVE: define specific error codes for all errors
+	parserErrorCode     = -32000
```

Comment:
Can you give this a more concrete name like `evmExecutionReverted`? https://stackoverflow.com/questions/71296510/code-32000-message-execution-reverted

Not sure if it's correct, but just the first thing that came up

Reply:
[According to this it's:](https://ethereum-json-rpc.com/errors)
```
-32000
Invalid input
Missing or invalid parameters
```

But more specifically it seems there 32... error codes an EVM specific. So my question is do we want to be using this for an error the emerges before we even get to the point of selecting a specific service? Using 32... is a holdover from Middleware and I we agree that we should not bring techdebt from Middleware to PATH>

IMO we shouldn't and a simple http 400 status code should suffice here. WDYT?

Also CCing @adshmh for your thoughts on error codes in PATH generally.

Reply:
Outdated, resolving.